README.md

```diff
@@ -14,7 +14,7 @@ files for assignments that require GatorGradle. ConfiGator uses
 
 ## Usage
 
-### Installing Python dependencies
+### Installing Python Dependencies
```

Comment:
This section should be relegated to a "contributing section" or removed altogether, and instructions on installing your program from PyPI (using `pipx` or `pip`) should be included instead.